config/manager/kustomization.yaml

```diff
@@ -6,8 +6,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ghcr.io/kubestellar/ocm-status-addon
-  newTag: 0.2.0-alpha.1
+  newName: ko.local/ocm-status-addon
+  newTag: b8c4f00
```

Comment:
What is this tag ?  Is this a commit number ?

Reply:
I think the intent is that `make chart` overwrites this, see the `$(KUSTOMIZE) edit` in that recipe?

Reply:
It would be comforting if the value here were obviously a dummy value, such as `replaceme`.

Reply:
yes, that gets updated by Kustomize. Will put a placeholder there.